cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra before FDR
+def: "Number of identified spectra the identification process provides." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000070
-name: MS2 Total ion current chromatogram
-def: "The total ion current chromatogram of MS2 measurements. The first column contains the retention time values in second, the second column the corresponding relative intensities." [PSI:QC]
-comment: The metric enables to check visually e.g. MS2 TIC absolute value compared to MS1 total ion chromatogram or other analyses MS2 TICs.
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-is_a: MS:4000024 ! MS2 metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000067 ! Total ion current chromatogram
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra after FDR
+def: "Number of identified spectra at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000071
-name: Fraction of peptide IDs around max MS1
-def: "Fraction of all peptides identified at least X minutes earlier or later than the most intense MS1 signal of the identification. The first column contains the time in minute-wise steps of seconds, the other comlumn the fraction of identified. Negative time values indicate minutes before and positive after maximum MS1." [PSI:QC]
-comment: Estimates very early peak broadening.
-is_a: MS:4000006 ! table
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000239 ! Fraction
+name: number of identified peptides
+def: "Number of identified peptides at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000072
-name: Interquartile RT period for peptide identifications
-def: "The interquartile retention time period, in seconds, for all peptide identifications over the complete run." [PSI:QC]
-comment: Longer times indicate better chromatographic separation.
+name: number of identified distinct peptides
+def: "Number of identified peptides without duplicate count (or modifications) at given FDR threshold." [PSI:QC]
```

Comment:
overlapping peptides are counted separately, I assume, e.g. a missed cleavage is a new peptide?!